test/integration/enterprises.js

```diff
@@ -138,7 +138,7 @@ describe('test/integration/enterprises Enterprises API', () => {
       .query({ detailed : 1 })
       .then(res => {
         helpers.api.listed(res, numEnterprises);
-        expect(res.body[0]).to.have.all.keys(responseKeys);
+        expect(res.body[0]).to.have.all.keys([...responseKeys, 'dhis2_uid']);
```

Comment:
Why not add 'dhis_uid' to the response keys?

Reply:
It only is applicable in the `detailed` query, so I didn't bother.